src/apis/festivalManager/getMyFestivalPermissionDetail.ts

```diff
@@ -0,0 +1,25 @@
+import { apiInstance } from '@/apis/apiInstance';
+import API_ENDPOINTS from '@/constants/apiEndpoints';
+import { generatePath } from 'react-router-dom';
+import type { AxiosResponse } from 'axios';
+import type { ApiResponseItem } from '@/apis/apiResponse';
+import type { ApplicationState } from '@/apis/apiResponse';
+
+export interface FestivalPermissionDetail {
+  id: number;
+  title: string;
+  posterImg: string;
+  appliedDate: string;
+  state: ApplicationState;
+  docs: string[];
+}
```

Comment:
```suggestion
export interface FestivalPermission & {
  posterImg: string;
  appliedDate: string;
  docs: string[];
}
```

이런 형태로 중복된 타입을 인터섹션해서 관리하면 깔끔할 것 같습니다.

Reply:
FestivalPermissionItme 이라는 타입을 말씀하신거겠죠? 이미 extends문법을 활용한 적이 있어서 해당 방식으로 해놓을게요

Reply:
넵 맞습니다!